tensorflow/python/ops/init_ops.py

```diff
@@ -95,7 +95,10 @@ def constant_initializer(value=0, dtype=dtypes.float32):
 
   Returns:
     An initializer that generates tensors with constant values.
-
+    
+  Raises:  
```

Comment:
This is explained above I think...

"If the total number of elements in `value` is greater than the number of elements required by the tensor shape...."

so perhaps you could say something like...

Raises:
  ValueError: When total number of elements in `value` is more than the number of elements required by the tensor shape as explained above.